.gitignore

```diff
@@ -10,6 +10,8 @@ yarn-debug.log*
 yarn-error.log*
 lerna-debug.log*
 
+yarn.lock
```

Comment:
hmm. shouldn't the lock files be in git to define exact current dependencies?

Reply:
oh well - ur right

Reply:
removed it in https://github.com/bumi/lightning-browser-extension/pull/41/commits/f7e4da5f5452bf72b1a39ee6634f04287fc27ee0